src/assets/icons/ArrowRight.tsx

```diff
@@ -0,0 +1,21 @@
+import { forwardRef } from 'react';
+
+import { Icon, IconProps } from '@/component-library/Icon';
+
+const ArrowRight = forwardRef<SVGSVGElement, IconProps>((props, ref) => (
```

Comment:
Most likely we will have an `ArrowRight`, so I would name this one `ArrowRightCircle`.